extensions/MongoDbAtlas/MongoDbAtlas/MongoDbAtlasStorage.cs

```diff
@@ -105,7 +107,8 @@ public Task DeleteDocumentDirectoryAsync(string index, string documentId, Cancel
                 {
                     { "index", index },
                     { "documentId", documentId },
-                    { "fileName", fileName }
+                    { "fileName", fileName },
+                    { "contentType", "application/octet-stream"}
```

Comment:
We should reuse mime detection, the approach used in other content storage classes, deriving the type from the file name 

e.g. see https://github.com/microsoft/kernel-memory/blob/dcd2a0e9c5dfeb1f896f746f21272673f71a4109/extensions/AzureBlobs/AzureBlobsStorage.cs#L196

Reply:
Thanks! Done!